src/commands/ext-install.ts

```diff
@@ -83,6 +84,12 @@ export default new Command("ext:install [extensionName]")
     }
     let source;
     let extVersion;
+
+    // TODO(b/220900194): Track url install usage to determine if we can remove this feature.
```

Comment:
Isn't this PR completing this TODO?

Reply:
Updated this to: Remove tracking of url install once we remove this feature.